tools/dotnet-linker/SetupStep.cs

```diff
@@ -42,6 +42,14 @@ protected override void Process ()
 				Console.WriteLine ("Pipeline Steps:");
 				foreach (var step in Steps) {
 					Console.WriteLine ($"    {step}");
+					if (step is SubStepsDispatcher) {
+						var substeps = typeof (SubStepsDispatcher).GetField ("substeps", BindingFlags.Instance | BindingFlags.NonPublic)?.GetValue (step) as IEnumerable<ISubStep>;
```

Comment:
Do we need to be worries that this non-public instance field gets refactored out from under us?

Reply:
No, the worst thing that will happen is that we won't get this additional debug info in verbose mode.

Reply:
That's fair. It would be nice if we had a test that told us that happened, else we're likely to not notice. Though, the whole thing is low probability and low impact, so :shrug: